code/modules/mob/living/say.dm

```diff
@@ -85,12 +85,14 @@ proc/get_radio_key_from_channel(var/channel)
 		verb = "stammers"
 		speech_problem_flag = 1
 
-	if(GREY in mutations)
-		message = "<span class='grey'>[message]</span>"
-
-	else if(COMIC in mutations)
+	if(COMIC in mutations)
 		message = "<span class='sans'>[message]</span>"
 
+	else if(ishuman(src))

```

Comment:
This needs to go under `/mob/living/carbon/human/handle_speech_problems`, `human/say.dm, 105`